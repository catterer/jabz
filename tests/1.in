[
    ["reqid_string_100002"],
    [
        1231,
        122
    ],
    3, ["value1"],
    4, ["value2"],
    {"u64" : 123123123}
]
