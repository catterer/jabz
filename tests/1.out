00000000  47 00 00 00 13 00 00 00  72 65 71 69 64 5f 73 74  |G.......reqid_st|
00000010  72 69 6e 67 5f 31 30 30  30 30 32 08 00 00 00 cf  |ring_100002.....|
00000020  04 00 00 7a 00 00 00 03  00 00 00 06 00 00 00 76  |...z...........v|
00000030  61 6c 75 65 31 04 00 00  00 06 00 00 00 76 61 6c  |alue1........val|
00000040  75 65 32 b3 b5 56 07 00  00 00 00                 |ue2..V.....|
0000004b
